packages/loot-core/src/types/api-handlers.d.ts

```diff
@@ -23,6 +25,8 @@ export interface ApiHandlers {
     password?: string;
   }) => Promise<void>;
 
+  'api/get-budgets': () => Promise<(Budget | APIRemoteFileEntity)[]>;
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏any chance we could transform the `Budget` models to `APIRemoteFileEntity`? Via `toExternal()`
```suggestion
  'api/get-budgets': () => Promise<APIRemoteFileEntity[]>;
```

Reply:
How about the current change?